Makefile

```diff
@@ -463,12 +464,11 @@ podman-remote-%-release:
 	rm -f release.txt
 	$(MAKE) podman-remote-release-$*.zip
 
-BINDINGS_SOURCE = $(wildcard pkg/bindings/**/types.go)
-.generate-bindings: $(BINDINGS_SOURCE)
+.PHONY: geenrate-bindings
```

Comment:
Aw, man, sorry to do this, but that's a typo

Reply:
Too many things in parallel ... sorry for my sloppiness